packages/mobile/src/screens/edit-track-screen/components/CollectibleGatedAvailability.tsx

```diff
@@ -181,7 +181,12 @@ export const CollectibleGatedAvailability = ({
       setTrackAvailabilityFields(
         {
           is_premium: true,
-          premium_conditions: { nft_collection: selectedNFTCollection },
+          premium_conditions: selectedNFTCollection
```

Comment:
hmm this might need to be
`premium_conditions: { type: PremiumContentType.COLLECTIBLE_GATED, nft_collection: selectedNFTCollection }`